### ISSUE TITLE
Changing timezones in the Steam Deck UI doesn't work

### ISSUE THREAD

**Issue by samueldr** [Jovian-Experiments/Jovian-NixOS#129]
Split off from https://github.com/Jovian-Experiments/Jovian-NixOS/issues/125.

TLDR: changing timezone in the UI does nothing.

Hint: look at the journalctl log, it will show that a dbus call fails the first time, then is not attempted any other times in that launch of the app.

~~This is most likely not a vendor issue, but something we are or are not doing.~~

Low priority since `config.time.timeZone` is so much more NixOS.

* * *

This could be "solved" by looking at if setting [`time.timeZone`](https://search.nixos.org/options?channel=unstable&show=time.timeZone&from=0&size=50&sort=relevance&type=packages&query=timezone) to `null` makes the imperative `timedatectl` semantics work.
